specification/monitor/resource-manager/Microsoft.Insights/stable/2019-06-01/actionGroups_API.json

```diff
@@ -352,16 +352,6 @@
           "readOnly": true,
           "description": "Azure resource type"
         },
-        "kind": {
-          "type": "string",
-          "readOnly": true,
-          "description": "Azure resource kind"
-        },
```

Comment:
We have other resource models but also have a kind property: 
https://github.com/Azure/azure-rest-api-specs/blob/36ac14c46828ae36365ca9d68c26d75d9ade48f0/specification/monitor/resource-manager/Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json#L353
(there are more)
but they do not have a name of `AzureResource`, I suppose maybe we could remove it now (considering removing identity is already a breaking change)

Reply:
> We have other resource models but also have a kind property:
> 
> https://github.com/Azure/azure-rest-api-specs/blob/36ac14c46828ae36365ca9d68c26d75d9ade48f0/specification/monitor/resource-manager/Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json#L353
> 
> 
> (there are more)
> but they do not have a name of `AzureResource`, I suppose maybe we could remove it now (considering removing identity is already a breaking change)

My team do not own those files, so I am not sure if I should be removing them. If you would like to remove these, I think we should reach out to the right owners to be safe.